internal/test/builder/bootstrap.go

```diff
@@ -0,0 +1,104 @@
+/*
```

Comment:
I'm not fully following, are all these files needed for envtest?

Reply:
you are absolutely right. Removed all that unnecessary code